### PR TITLE
feature/SPRIND-92

### PR DESCRIPTION
**Pull request by BtencateSphereon**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -63,20 +63,20 @@
...
-    "@sphereon/ssi-sdk.event-logger": "0.30.2-feature.SPRIND.61.JARM.pex.fixes.39",
-    "@sphereon/ssi-sdk.issuance-branding": "0.30.2-feature.SPRIND.61.JARM.pex.fixes.39",
-    "@sphereon/ssi-sdk.mdl-mdoc": "0.30.2-feature.SPRIND.61.JARM.pex.fixes.39",
-    "@sphereon/ssi-sdk.oid4vci-holder": "0.30.2-feature.SPRIND.61.JARM.pex.fixes.39",
-    "@sphereon/ssi-sdk.sd-jwt": "0.30.2-feature.SPRIND.61.JARM.pex.fixes.39",
-    "@sphereon/ssi-sdk.siopv2-oid4vp-op-auth": "0.30.2-feature.SPRIND.61.JARM.pex.fixes.39",
-    "@sphereon/ssi-sdk.vc-handler-ld-local": "0.30.2-feature.SPRIND.61.JARM.pex.fixes.39",
-    "@sphereon/ssi-sdk.xstate-machine-persistence": "0.30.2-feature.SPRIND.61.JARM.pex.fixes.39",
-    "@sphereon/ssi-types": "0.30.2-feature.SPRIND.61.JARM.pex.fixes.39",
+    "@sphereon/ssi-sdk.agent-config": "0.30.2-feature.SPRIND.92.149",
```
Comment (sanderPostma) [inline package.json]: Don't forget to updated after https://github.com/Sphereon-Opensource/SSI-SDK/pull/259 is merged

File: src/migrations/1730209432321-AddTrustAnchorContacts.ts [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+                                           'IssuerLocaleBranding',
+                                           datetime('now'), 
+                                           datetime('now')
+                                           )`,
+    );
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    // TODO implement migration down
+    return Promise.reject(Error('Not yet implemented'));
```
Comment (sanderPostma) [inline src/migrations/1730209432321-AddTrustAnchorContacts.ts:266]: This is something AI can do flawlessly
```
import { QueryRunner } from 'typeorm';

export class AddTrustAnchorContacts1730209432321 implements MigrationInterface {
  // ... existing up() method remains the same ...

  public async down(queryRunner: QueryRunner): Promise<void> {
    // Delete Findynet Federation related records
    await queryRunner.query(
      `DELETE FROM "BaseLocaleBranding" WHERE id = '8c66b536-eb30-45f8-8ed0-2f4a2203c16f'`
    );
    
    await queryRunner.query(
      `DELETE FROM "ImageAttributes" WHERE id = '154f90c0-2135-439e-b23d-458e3d781589'`
    );
    
    await queryRunner.query(
      `DELETE FROM "ImageDimensions" WHERE id = 'e17767fc-91b8-4546-af34-91305c89571d'`
    );
    
    await queryRunner.query(
      `DELETE FROM "IssuerBranding" WHERE id = 'e6523745-aa00-43ee-8c92-d352243c64b1'`
    );
    
    await queryRunner.query(
      `DELETE FROM "CorrelationIdentifier" WHERE id = 'd56bb8f2-db52-4923-a4ff-9e835e16290c'`
    );
    
    await queryRunner.query(
      `DELETE FROM "Identity" WHERE id = '8cea5ccd-59bf-4c72-9844-ddfcfdf6ca56'`
    );
    
    await queryRunner.query(
      `DELETE FROM "BaseContact" WHERE id = '7e585fee-7f7e-469d-8c99-96a7ec3e60e4'`
    );
    
    await queryRunner.query(
      `DELETE FROM "Party" WHERE id = '7f16a97f-19e8-4387-bcac-a23f82797f8f'`
    );

    // Delete Sphereon Federation related records
    await queryRunner.query(
      `DELETE FROM "BaseLocaleBranding" WHERE id = 'd969e748-1ae7-45df-8638-a716350869d5'`
    );
    
    await queryRunner.query(
      `DELETE FROM "ImageAttributes" WHERE id = '19803a99-8d0a-4fe8-8565-278032b7903b'`
    );
    
    await queryRunner.query(
      `DELETE FROM "ImageDimensions" WHERE id = '3f38b5b9-bf7c-4eab-91c4-6970578b9d3a'`
    );
    
    await queryRunner.query(
      `DELETE FROM "IssuerBranding" WHERE id = '999ae111-6d66-46da-ac01-858d1df87a83'`
    );
    
    await queryRunner.query(
      `DELETE FROM "CorrelationIdentifier" WHERE id = '2b4130d5-bc01-4428-9f8e-8ff1e536f666'`
    );
    
    await queryRunner.query(
      `DELETE FROM "Identity" WHERE id = '66b16b8c-cf94-449c-820d-fd888dec7a88'`
    );
    
    await queryRunner.query(
      `DELETE FROM "BaseContact" WHERE id = '0014e6b8-9271-4d4a-939c-dd1b23880f4d'`
    );
    
    await queryRunner.query(
      `DELETE FROM "Party" WHERE id = '654d698e-fd26-4ac7-8098-7e14870140df'`
    );
  }
}
```

**Review by sanderPostma — CHANGES_REQUESTED** [review]

**Review by sanderPostma — APPROVED** [review]
LGTM
the comments are FYI

File: src/migrations/1730209432321-AddTrustAnchorContacts.ts (hunk 2 above)
Comment (BtencateSphereon) [reply src/migrations/1730209432321-AddTrustAnchorContacts.ts:266]: hmmm yes, but these are temp migrations just so for now i am leaving it. but sure, easy enough for an AI

**Pull request merged**